src/DynamoCoreWpf/Views/Core/DynamoView.xaml

```diff
@@ -113,9 +113,10 @@
         <KeyBinding Key="O"
                     Modifiers="Control"
                     Command="{Binding Path=DataContext.ShowOpenDialogAndOpenResultCommand, RelativeSource={RelativeSource FindAncestor, AncestorType={x:Type controls:DynamoView}}}" />
-        <KeyBinding Key="T"
-                    Modifiers="Control"
-                    Command="{Binding Path=DataContext.ShowOpenTemplateDialogCommand, RelativeSource={RelativeSource FindAncestor, AncestorType={x:Type controls:DynamoView}}}" />
```

Comment:
Curious what is this combo conflicting with?

Reply:
Something that I could not track interestingly, online sources say its either a conflict or a binding issue, there is no problem in the binding since it is working with other keys, so the only explanation could be a conflict imo.